target/linux/mediatek/dts/mt7623a-unielec-u7623-02.dtsi

```diff
@@ -121,26 +121,24 @@
 		#address-cells = <1>;
 		#size-cells = <0>;
 
-		mt7530: switch@0 {
+		mt7530: switch@1f {
 			compatible = "mediatek,mt7530";
+			reg = <31>;
+			#address-cells = <1>;
+			#size-cells = <0>;
+			pinctrl-names = "default";
+			mediatek,mcm;
+			resets = <&ethsys 2>;
+			reset-names = "mcm";
+			core-supply = <&mt6323_vpa_reg>;
+			io-supply = <&mt6323_vemc3v3_reg>;
+
+			dsa,mii-bus = <&mdio>;
 		};
 	};
 };
 
 &mt7530 {
-	compatible = "mediatek,mt7530";
```

Comment:
Please keep the definition here, I think the switch configuration should be defined at one place. 
The change to 31 looks fine.

Reply:
Done